app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -76,6 +77,7 @@ fun SimpleTopAppBar(
 @Composable
 fun BottomAppBarAll(
     navController: NavController = rememberNavController(),
+    appSettings: AppSettings? = null,
```

Comment:
Since showBottomNav is the only one that's necessary, just pass that in rather than all the settings.